### DISCUSSION TITLE
Thank you note

### DISCUSSION THREAD

**Discussion by agilebean** [karloskalcium/md2enex#8] [General]
I'm sorry to file an issue but there was nowhere else I could leave this:

Just wanted to say how grateful I am for this repo to the author.

My use case is to convert chatgpt output containing tables, and my one-and-only productivity tool is Evernote.
So I need to store all the info in Evernote, and with markdown tables it wasn't possible - until I saw this repo.

THANK YOU.

**Comment by karloskalcium** [comment]
@agilebean thanks for the feedback, I'm glad you found it useful, I changed your thank-you issue into a discussion here instead.
